metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -449,6 +509,11 @@ class TableauConfig(
         description="When enabled, sites are added as containers and therefore visible in the folder structure within Datahub.",
     )
 
+    access_role_ingestion: Optional[AccessRolesIngestionConfig] = Field(
+        default=None,
+        description="Settings for configuration of access roles ingestion.",
```

Comment:
Please update description as `Configuration settings for ingesting access roles. This allows Tableau groups to be imported as DataHub Roles.`